### ISSUE TITLE
Can page numbering be added?

### ISSUE THREAD

**Issue by tbrowder** [pod-to-pdf/Pod-To-PDF-Lite-raku#2]
The capability to add page numbering would be very handy for this light-weight module, if possible. If so, one of the styles I use most often for coordination documents being passed around (in person or via email) is, in the lower-right corner of each page:

    Page N of M

If it is not possible, how could I accomplish that?

Thanks, David!

**Comment by dwarring** [comment]
Hi Tom,
I'm looking at a secondary issue handling options. See [Pod::To::Xxxx options processing support](https://github.com/Raku/user-experience/issues/47)

**Comment by dwarring** [comment]
I'm going with custom options parsing, which looks easy enough in Raku:
```
        for @*ARGS {
            when /^'--page-numbers'$/  { $page-numbers = True }
            when /^'--width='(\d+)$/   { $width  = $0.Int }
            when /^'--height='(\d+)$/  { $height = $0.Int }
            when /^'--save-as='(.+)$/  { $pdf-file = $0.Str }
            default { note "ignoring $_ argument" }
        }
```

**Comment by tbrowder** [comment]
Good, that’s the way I like it. I thought there were lots of things that can be done with the right handle. Thanks.

**Comment by dwarring** [comment]
Hi Tom,
I've released Pod::To::PDF::Lite 0.1.5 that has several new options, including page numbers.

From the command line: `raku --doc=PDF::Lite myfile.pod6 --page-numbers`.

BTW, you can now also specify an output file:

 `raku --doc=PDF::Lite myfile.pod6 --page-numbers --save-as=myfile.pdf`

**Comment by tbrowder** [comment]
It works fine. This issue is closed.
